src/App.js

```diff
@@ -1,10 +1,14 @@
 import './App.css';
 import { RouterProvider } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import router from './router';
+import store from './redux/store';
 
 function App() {
   return (
-    <RouterProvider router={router} />
+    <Provider store={store}>
+      <RouterProvider router={router} />
+    </Provider>
```

Comment:
- [ ] Kindly consider wrapping the main component in <Provider> instead  because It is an easy way to make React component aware of the store and rerender when we change the store and consider passing the `store`  into the `<Provider>` as well 
- here is a hint 👇  :
```
import React from "react"
import ReactDOM from "react-dom/client"
import { Provider } from "react-redux"
import store from "./redux/store"

ReactDOM.render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById("root")
)

```